<launch>
    <include file="$(find ros_bagger)/launch/realsense.launch"/>
    <arg name="topics"                          default="[scan, tf, tf_static, map, odom, move_base_node/local_costmap/robot_footprint, move_base_node/current_goal, move_base_node/MIRPlannerROS/global_plan, move_base_node/MIRPlannerROS/local_plan]" />
    <arg name="trigger_timeout"                 default="5" />
    <arg name="trigger_topic"                   default="people_filter/people_tracker_measurements" />
    <arg name="topic_when_triggered"            default="camera/color/image_raw" />
    <node pkg="ros_bagger" type="bagger.py" name="ros_bagger" output="screen" >
        <rosparam param="topics" subst_value="True">$(arg topics)</rosparam>
        <param name="trigger_timeout"               value="$(arg trigger_timeout)" />
        <param name="trigger_topic"                 value="$(arg trigger_topic)" />
        <param name="topic_when_triggered"          value="$(arg topic_when_triggered)" />
    </node>
</launch>
